app/models/agents/typeform_agent.rb

```diff
@@ -32,7 +32,7 @@ class TypeformAgent < Agent
         * `comment_question_ids` - Hard-code he comma separated list of ids of the comment questions (agent will pick the first one present) if `guess_mode` is off
         * `expected_receive_period_in_days` - Specify the period in days used to calculate if the agent is working.
         * `limit` - Number of responses to fetch per run, better to set to a low number nad have the agent run more often.
-    MD
+        * `offset` - Number of responses to offset by if you want to go back in the past
 
     event_description <<-MD
       Events look like this:
```

Comment:
MD deleted, the description looks bad.

Reply:
Oh right, can you pls fix via a hotfix commit on master